CRM Solution Manager/CRM Solution/CrmSolutionHelper.cs

```diff
@@ -679,16 +679,23 @@ private void ExportSolution(OrganizationServiceProxy serviceProxy, SolutionFileI
         /// </summary>
         /// <param name="serviceProxy">organization service proxy</param>
         /// <returns>returns entity collection</returns>
-        private EntityCollection FetchSourceControlQueues(OrganizationServiceProxy serviceProxy)
+        private EntityCollection FetchSourceControlQueues(OrganizationServiceProxy serviceProxy, string mode)
         {
             QueryExpression querySampleSolution = new QueryExpression
             {
                 EntityName = Constants.SounceControlQueue,
                 ColumnSet = new ColumnSet() { AllColumns = true },
                 Criteria = new FilterExpression()
             };
-
-            querySampleSolution.Criteria.AddCondition(Constants.SourceControlQueueAttributeNameForStatus, ConditionOperator.Equal, Constants.SourceControlQueueQueuedStatus);
+            if (mode == "WEB")
```

Comment:
move all this to constants file, this can be in multiple places

also use below to have column names for the constants
![image](https://user-images.githubusercontent.com/35560136/62406575-c3d4c300-b5cb-11e9-9fa8-2835c194012e.png)